docs/userman/advanced_features/3rd party integration/databricks.md

```diff
@@ -0,0 +1,197 @@
+- _**write_query_parameter**_ represents parameters to be used when executing the write query. Default is None
+- _**append_query_parameter**_ represents parameters to be used when executing the append query. Default is None
+- _**profile**_ represents additional options of Databricks credential profile to be used when reading
+    and writing to the table.<br/>
+- _**cluster_id**_ represents additional options of the ID of Databricks cluster to be used when reading
+    and writing to the table.<br/>
+- _**exposed_type**_ indicates the data type returned when reading the data node (more
+    examples of reading from a Databricks Table data node with different *exposed_type* are available
+    in the [read/write a data node](data-node-usage.md#databricks-table) documentation):
+        - By default, *exposed_type* is "spark", and the data node reads the Databricks table
+            as a Spark DataFrame (`pyspark.sql.dataframe.DataFrame`) when executing the read method.
+        - If the *exposed_type* provided is "pandas", and the data node reads the Databricks table
+            as a Pandas DataFrame (`pandas.DataFrame`) when executing the read method.
+        - If the *exposed_type* provided is "numpy", the data node reads the Databricks table
+            as a NumPy array (`numpy.ndarray`) when executing the read method.
+        - If the *exposed_type* provided is "polars", and the data node reads the Databricks table
+            as a Polars DataFrame (`polars.DataFrame`) when executing the read method.
+
+To configure a Databricks SQL data node, use the `configure_databricks_sql_data_node` method:
+
+```python linenums="1"
+{%
+include-markdown "./code-example/data-node-config/data-node-config_sql_databricks.py"
+comments=false
+%}
+```
+
+In lines 33-41, we configure a basic Databricks SQL data node with the id "historical_temperature".
+The data node will use the query "SELECT * FROM hist_temp" to read data from Databricks database.
+It will use "build_hist_temp_write_query" function and "build_hist_temp_append_query" function
+to build the write and append queries to write and append data to Databricks database.
+Its *scope* is by default `SCENARIO`. The profile to be used is "default" profile with the cluster id is
+"0123-456789-dbscluster2". The exposed type of this data node will be the default value "spark".
+
+In lines 33-41, we configure a basic Databricks SQL data node with the id "historical_log".
+The data node will use the query "SELECT * FROM hist_log" to read data from Databricks database.
+It will use "build_hist_log_write_query" function and "build_hist_log_append_query" function
+to build the write and append queries to write and append data to Databricks database.
+Its *scope* is by default `SCENARIO`. The profile to be used is "default" profile with the cluster id is
+"0123-456789-dbscluster2". The exposed type of this data node will be the default value "polars".
+
```

Comment:
Can we add one or two lines explaining the usage is the same as a normal data node. 
read/write/append, etc... Why not a short example.

What do you think?